extern/open.php

```diff
@@ -11,6 +12,11 @@
   exit();
 }
 
+if (!CRM_Utils_Rule::positiveInteger($queue_id)) {
```

Comment:
yeah - or use retrieveValue rather that $_GET directly

Reply:
yeh maybe i dunno however this just felt sensible way of managing it

Reply:
The reason I prefer to go with a retrieve fn variant is we should be able to grep our code base for instances of $_GET & find very few - so eliminating them makes future audits easier

Reply:
have switched it to that now hopefully @eileenmcnaughton